tests/saw/spec/handshake/rfc_handshake_tls13.cry

```diff
@@ -305,3 +305,61 @@ rfc2S2N msg = mkAct recordType messageType writer
                  | msg.sender == both then 'B'
                  else '!'
         noMessageType = TLS_HELLO_REQUEST // HelloRequests don't exist in tls1.3
+
+/**
+ * printHandshake and s2nToWords are two helper functions for debugging cryptol
+ * Take the counterexample from tls13rfcSimulatesS2N and plug the values into
+ * testParameters and testConnection. Run printHandshake with ascii set to on. 
+ * Note with ascii on, you may see actual_protocol_version = '"'. This is because
+ * decimal value 34 (hex value 22) == asci character double quotes. 
+ */
+
+testParameters : Parameters
+testParameters = {
+            compat_mode = [False, True],
+            psk_mode = False,
+            retry = False,
+            client_auth = True,
+            no_client_cert = True,
+            zero_rtt = False
+        }
+
+testConnection : connection
+testConnection = {
+            handshake = {handshake_type = 0x00000000,
+                message_number = 0x00000000},
```

Comment:
Even if it did, I'm not sure we'd want to use it. If we did, everyone who wanted to use the debug functions would have to swap out the default initializer for an explicit initialization. It's probably more helpful for developers to just make anyone who adds a new field also add it here.

Reply:
I don't really know what you mean "swap out .. for an explicit initialization". This very code:

```
handshake = {handshake_type = 0x00000000,
                message_number = 0x00000000},
```

is reused in several places. Why can't we say `defaulHandshake`. Then update the single defaultHandshake? Or, since this is all very functional, `getDefaultHandshake -> handshake` ?